samples/BikeSharingApp/Bikes/charts/bikes/values.yaml

```diff
@@ -5,7 +5,7 @@ fullnameOverride: bikes
 replicaCount: 1
 image:
   repository: azdspublic/bikesharing-bikes
-  tag: build.20200619.1
+  tag: 20210331.1
```

Comment:
Could you please clarify why we don't need the `build` anymore?

Reply:
OK thanks. We might as well just go back to using the `latest` tag everywhere in Bike Sharing. As at this point Bike Sharing is about to be abandoned as a sample, and only used by CTI or when testing ourselves, we should try to make this as low maintenance as possible.

Reply:
I like that idea - that way we don't have to keep updating the values file with the latest tag and the build pipeline can take care of building and pushing the latest code with latest tag whenever there are any changes.
I'll update the bug with these details. Thanks!